interfaces/api/dependencies.py

```
@@ -427,8 +435,8 @@ def get_embedding_service():
             logger.info(f"Using local embedding service: {model_path}, GPU: {use_gpu}")
             return LocalEmbeddingService(model_name=model_path, use_gpu=use_gpu)
         elif service_type == "openai":
-            if not os.getenv("OPENAI_API_KEY"):
-                logger.warning("EMBEDDING_SERVICE=openai 但 OPENAI_API_KEY 未设置，向量检索已禁用")
+            if not (os.getenv("EMBEDDING_API_KEY") or os.getenv("OPENAI_API_KEY")):
+                logger.warning("EMBEDDING_SERVICE=openai 但 EMBEDDING_API_KEY/OPENAI_API_KEY 未设置，向量检索已禁用")
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)